recipes/protobuf/all/conanfile.py

```diff
@@ -176,9 +176,10 @@ def _patch_sources(self):
                  "string(REPLACE \";\" \":\" CUSTOM_DYLD_LIBRARY_PATH \"${CUSTOM_DYLD_LIBRARY_PATH}\")\n"
                  "add_custom_command(")
             )
+            cmd_str = "COMMAND  protobuf::protoc" if tools.Version(self.version) < "3.20.0" else "COMMAND protobuf::protoc"
```

Comment:
🤔 just an extra blank? is there any black magic involved here?

Reply:
Yep, the extra blank got removed in v3.20.0.